temporalio/bridge/worker.py

```diff
@@ -33,13 +33,15 @@ class WorkerConfig:
     max_outstanding_workflow_tasks: int
     max_outstanding_activities: int
     max_outstanding_local_activities: int
-    max_concurrent_wft_polls: int
+    max_concurrent_workflow_task_polls: int
     nonsticky_to_sticky_poll_ratio: float
-    max_concurrent_at_polls: int
+    max_concurrent_activity_task_polls: int
```

Comment:
I ended up not exposing these in TS, core will never poll activities unless `lang` has an active poll request so effectively this param is not used and for workflows this used to be the case, I'm not entirely sure what the new behavior is after the huge core refactor.
Eventually we should autoscale pollers or better yet use grpc streaming for better latency.

Reply:
:+1: May end up removing.